notifique/src/main/res/layout/list_item_children.xml

```diff
@@ -5,11 +5,21 @@
       android:layout_height="wrap_content"
       android:orientation="horizontal"
       >
+    <ImageView
+        android:id="@+id/icon_picture"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:adjustViewBounds="true"
+        android:maxWidth="42dp"
+        android:maxHeight="42dp"
+        android:contentDescription="@string/notifique_icon_content_description"
+        />
     <TextView
         android:id="@+id/app_name"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
+        android:layout_weight="2"
```

Comment:
why?

Reply:
reverted - probably forgot to change back when I was trying to fix the listitems